apps/music/style/music.css

```diff
@@ -555,13 +555,6 @@ html[dir="rtl"] .tile:nth-child(12n+6) {
   right: 0.5%;
 }
 
-/* Favor scrolling performance over quality while scrolling */
-#views.scroll li.list-item,
```

Comment:
@davidflanagan Yes, I think that's a typo in https://bugzilla.mozilla.org/show_bug.cgi?id=978065 and no one found it until now... anyway we don't need it so this should be fine.